__tests__/app.test.js

```diff
@@ -21,6 +21,14 @@ describe("Sad Paths", () => {
         expect(msg).toBe("Invalid path");
       });
   });
+  test("status 400: responds for bad requests", () => {
```

Comment:
This sad path test should be under the describe block on line 54, keep your sad path testing with the endpoint it relates to - not separate blocks

Reply:
Added more tests and transferred to the right places